src/libraries/System.Runtime/tests/System.Runtime.CompilerServices.Unsafe.Tests/UnsafeTests.cs

```diff
@@ -609,6 +609,9 @@ public static void RefSubtract()
 
             ref string r3 = ref Unsafe.Subtract(ref r2, 3);
             Assert.Equal("abc", r3);
+
+            static ref byte NullTest(nuint offset) => ref Unsafe.Subtract(ref Unsafe.NullRef<byte>(), offset);
```

Comment:
Add a noinlining attribute here, otherwise it will fold it before it gets to that morph opt?